pkgs/top-level/all-packages.nix

```diff
@@ -18577,7 +18577,7 @@ with pkgs;
   };
 
   weasis = callPackage ../by-name/we/weasis/package.nix {
-    jre = jdk21;
+    jre = jdk23;
```

Comment:
The package refuses to compile with JDK 21 (claiming that JDK 21 is too old) and JDK 22 has reached end-of-life, so JDK 23 is the only thing that works.

Reply:
Upstream do explicitly say it needs jdk 23+: https://weasis.org/en/getting-started/building-weasis/